### DISCUSSION TITLE
Config file for NAS

### DISCUSSION THREAD

**Discussion by templeowls21** [blakeblackshear/frigate#14289] [Ask A Question]
Very stupid question I'm sure, but I can't seem to find the answer.

I am using the Frigate add-on in HA and have mounted my new NAS.

What do I specifically put in the config file for recordings to drop to the correct folder/NAS?

**Comment by NickM-27** [comment]
https://docs.frigate.video/guides/ha_network_storage

**Reply by templeowls21** [reply]
So I’ve been doing that but it’s not working. Still storing the recordings in my server’s SSD.

I think I’m doing the “database:
  path: /config/frigate.db” piece wrong. I put that line in my frigate.yaml file but I’m I supposed to be doing something else or beyond that? Sorry; all new to me

**Reply by NickM-27** [reply]
Are you sure it's not recording on the NAS? The folder will still show up on the media folder but it is a mirror and the files are not stored on the SSD

**Reply by templeowls21** [reply]
100%. Nothing is going into the nas and the server SSD slowly gets bigger

**Reply by NickM-27** [reply]
can you show an example of this?

**Comment by templeowls21** [comment]
![Screenshot 2024-10-14 081652](https://github.com/user-attachments/assets/331bbced-7ceb-4dcd-89e7-ac34e1380156)
See attached of my server filling up (also NAS is included). 

My steps are:
- turn off Frigate add on
- delete frigate folder in 'media' via samba
- add NAS and title it frigate
- restart HA
- start Frigate add-on

**Please note:** when I go to my 'media' folder in HA, there's the Frigate folder with the Frigate logo where my recordings are being saved on my SSD and then there's a folder named "my media" where my NAS Frigate folder is going. Not sure if that's the issue

**Reply by NickM-27** [reply]
did you set target as /media/frigate ?

**Reply by templeowls21** [reply]
here's my NAS settings (if this is what you're asking):

![Screenshot 2024-10-14 083601](https://github.com/user-attachments/assets/de716301-cfb8-4895-b354-8a7b8cb955bf)

Please note: when I go to my 'media sources' in HA, there's the Frigate folder with the Frigate logo where my recordings are being saved on my SSD and then there's a folder named "my media" where my NAS Frigate folder is going. Not sure if that's the issue

**Reply by NickM-27** [reply]
you should try `frigate` not `Frigate`

**Reply by templeowls21** [reply]
Gotcha, I'll give that a shot when I get home. Thanks!!

**Reply by templeowls21** [reply]
> you should try `frigate` not `Frigate`

That seems to have done the trick; I'm an idiot. Thanks so much!

**Reply by NickM-27** [reply]
Glad it's working 👍
